validator/client/validator.go

```diff
@@ -665,6 +667,11 @@ func (v *validator) logDuties(slot types.Slot, duties []*ethpb.DutiesResponse_Du
 	}
 	for i := types.Slot(0); i < params.BeaconConfig().SlotsPerEpoch; i++ {
 		if len(attesterKeys[i]) > 0 {
+			for _, truncKey := range attesterKeys[i] {
```

Comment:
I'm concern about this potential o(n^2) loop. Let's optimize it to o(n) if we can